### DISCUSSION TITLE
Mermaid doesn't apply dark theme when using github dark mode

### DISCUSSION THREAD

**Discussion by nathan-fiscaletti** [community/community#12116] [Accessibility] [answered]
I currently have the standard dark theme set on my github. However, when i create a mermaid diagram in one of my projects readmes, it displays like this:

![image](https://user-images.githubusercontent.com/1699763/155799402-92e2b7ba-7cd0-42e8-ac80-608e11a5ab04.png)

Here is how the same diagram displays when i use the [Mermaid Live Editor](https://mermaid.live/edit)

![image](https://user-images.githubusercontent.com/1699763/155799476-07d4fd83-9ebb-465b-b9df-be2a479915d3.png)

**Comment by nathan-fiscaletti** [comment]
After refreshing the page a few times, it's now displaying the dark theme as expected. Not sure what was going on there ?

**Reply by gaknoll** [reply]
@nathan-fiscaletti that's a weird bug but I'm glad it resolved itself. Let us know if it happens again!

**Comment by nathan-fiscaletti** [comment, marked as answer]
It looks like this was caused by the Dark Reader chrome extension. When enabled, even if i have the dark github theme selected in my github settings, it will render the Mermaid charts with their light theme. I had to disable Dark Reader specifically for GitHub and then refresh the page to solve the issue.

**Reply by gaknoll** [reply]
@nathan-fiscaletti that will definitely do it! Dark Reader usually ends up supporting new variables we introduce, but it is delayed.

**Comment by marwie** [comment]
Facing the same issue right now here too: https://github.com/orgs/community/discussions/35733

**Comment by cehfisher** [comment]
Hi @nathan-fiscaletti - thank you for providing this feedback! This issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/12116#discussioncomment-2277971). Could you please confirm and close out this issue? Thank you!
